watertap/unit_models/reverse_osmosis_base.py

```diff
@@ -370,6 +370,64 @@ def eq_flux_mass(b, t, x, p, j):
                     - prop_perm.conc_mass_phase_comp[p, j]
                 )
 
+        if self.config.transport_model == "SKK":
+
+            self.eq_flux_mass.deactivate()
+
+            self.reflect_coeff = Var(
+                initialize=1,
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="Reflection coefficient of the membrane",
+            )
+
+            self.alpha = Var(
```

Comment:
Maybe this one should only be created with SKK enabled (and not with sol-diff, since it would just be 0). Or maybe we do include for both transport models and alpha would just be 0.